frontend/src/metabase/core/components/EditableText/EditableText.tsx

```diff
@@ -42,6 +46,13 @@ const EditableText = forwardRef(function EditableText(
   const [submitValue, setSubmitValue] = useState(initialValue ?? "");
   const displayValue = inputValue ? inputValue : placeholder;
   const submitOnBlur = useRef(true);
+  const previousInitialValue = usePrevious(initialValue);
+
+  useLayoutEffect(() => {
+    if (initialValue && initialValue !== previousInitialValue) {
+      setInputValue(initialValue);
+    }
+  }, [initialValue, previousInitialValue]);
```

Comment:
Seems like `EditableText` doesn't update if the value it "tracks" was updated in another place
This hook should fix that

Reply:
why doesn't `useEffect` work?

Reply:
It works actually, I used `useLayoutEffect` because of all the `EditableText` animation, but on a second thought it doesn't really make sense here. Thanks for the sanity check